aspnet/fundamentals/app-state/sample/src/AppState/project.json

```diff
@@ -18,7 +18,8 @@
   },
 
   "frameworks": {
-    "dnx451": { }
+    "dnx451": { },
+    "dnxcore50": { }
```

Comment:
@robbert229 Can you remove `dnxcore50` from the Frameworks section of `project.json` as binary serialization is not supported in CoreCLR. ( Look [here](https://github.com/dotnet/coreclr/issues/1203) )
We want to to ensure that the sample in the Docs repo is not broken